src/c++/perf_analyzer/genai-perf/README.md

```diff
@@ -87,47 +87,50 @@ genai-perf --help
 ## Triton with TRT-LLM
 
 ```bash
-genai-perf -m llama-2-7b --concurrency 1 --service-kind triton --output-format trtllm
+genai-perf -m llama-2-7b --concurrency 1 --service-kind triton --backend trtllm
 ```
 
 ## Triton with vLLM
 
 ```bash
-genai-perf -m llama-2-7b --concurrency 1 --service-kind triton --output-format vllm
+genai-perf -m llama-2-7b --concurrency 1 --service-kind triton --backend vllm
 ```
 
 ## OpenAI Chat Completions Compatible APIs
 
 https://platform.openai.com/docs/api-reference/chat
 
 ```bash
-genai-perf -m llama-2-7b --concurrency 1 --service-kind openai --endpoint v1/chat/completions --output-format openai_chat_completions
+genai-perf -m llama-2-7b --concurrency 1 --service-kind openai --endpoint v1/chat/completions
 ```
 
 ## OpenAI Completions Compatible APIs
 
 https://platform.openai.com/docs/api-reference/completions
 
 ```bash
-genai-perf -m llama-2-7b --concurrency 1 --service-kind openai --endpoint v1/completions --output-format openai_completions
+genai-perf -m llama-2-7b --concurrency 1 --service-kind openai --endpoint v1/completions
 ```
 
 # Model Inputs
-GenAI-Perf supports model inputs from either the HuggingFace OpenOrca or
-CNN_DailyMail datasets or it can create synthetic input data. This is specified
-using the `--input-type` CLI option.
-
-When the dataset is coming from HuggingFace you can specify the following
-options:
-* `--dataset`: HuggingFace dataset to use for benchmarking.
+GenAI-Perf supports model input prompts from either synthetically generated inputs,
+or from the HuggingFace OpenOrca or CNN_DailyMail datasets. This is specified
+using the `--prompt-source` CLI option.
 
 When the dataset is synthetic you can specify the following options:
-* `--num-of-output-prompts`: The number of synthetic output prompts to generate
-* `--input-tokens-mean`: The mean number of tokens of synthetic input data.
-* `--input-tokens-stddev`: The standard deviation number of tokens of synthetic
+* `--num-prompts`: The number of unique prompts to generate
```

Comment:
Nit: Could you use periods or not use periods for all of the options?

Very minor, but the consistency would be nice.

Reply:
I added some periods